cves/CVE-2007-0404.yml

```diff
@@ -49,27 +49,39 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A script for compliling messages was made vulnerable because the inputs
+  submitted were not being checked to make sure that they were not malicious in nature.
+  The script job was to make OS level commands and if the inputs were not checked correctly
+  changes could to be made to the system if a targetted input is submitted as an arguememnt or in the .po file
+  with commands. If some of these commands are exectuted hackers can take over the system by giving themselves
```

Comment:
Typo: executed

Reply:
You need a comma here: _If some of these commands are executed,_

Reply:
Thanks will fix these